bellows/zigbee/application.py

```diff
@@ -48,6 +48,24 @@ def __init__(self, ezsp, database_file=None):
         await self._cfg(c.CONFIG_END_DEVICE_POLL_TIMEOUT, 60)
         await self._cfg(c.CONFIG_END_DEVICE_POLL_TIMEOUT_SHIFT, 6)
 
+        await self.add_endpoint(
+            output_clusters=[zigpy.zcl.clusters.security.IasZone.cluster_id]
+        )
+
+    async def add_endpoint(self, endpoint=1,
+                           profile_id=zigpy.profiles.zha.PROFILE_ID,
+                           device_id=0xbeef,
```

Comment:
Why is device_id _beef_ ?

Reply:
it is a `device_type` of the endpoint. Since we're not strictly a `DeviceType.IAS_ZONE` device I've choose some "random" value. But we can change it to something else.